### ISSUE TITLE
Drop Datafusion

### ISSUE THREAD

**Issue by aersam** [bmsuisse/lakeapi#54]
Datafusion does not support Json Reading nor is there a way to register a pyarrow.Table instance. therefore, we cannot enable json tests for datafusion

Also the docs are ... not as good as in polars and duckdb

**Comment by dominikpeter** [comment]
Yes, we can drop it. It has the best delta-rs support, but it would probably only make sense when working in Rust. 

Let's drop it.
